mkdocs/__main__.py

```diff
@@ -162,17 +168,21 @@ def build_command(clean, **kwargs):
 @click.option('-m', '--message', help=commit_message_help)
 @click.option('-b', '--remote-branch', help=remote_branch_help)
 @click.option('-r', '--remote-name', help=remote_name_help)
+@click.option('--page-type',
+              type=click.Choice(['user', 'org', 'project'], case_sensitive=False),
+              help=page_type_help)
 @click.option('--force', is_flag=True, help=force_help)
 @click.option('--ignore-version', is_flag=True, help=ignore_version_help)
 @common_config_options
 @click.option('-d', '--site-dir', type=click.Path(), help=site_dir_help)
 @common_options
-def gh_deploy_command(clean, message, remote_branch, remote_name, force, ignore_version, **kwargs):
+def gh_deploy_command(clean, message, remote_branch, remote_name, page_type, force, ignore_version, **kwargs):
     """Deploy your documentation to GitHub Pages"""
     try:
         cfg = config.load_config(
             remote_branch=remote_branch,
             remote_name=remote_name,
+            page_type=page_type,
             **kwargs
         )
```

Comment:
Because you added the `page_type` argument here, a bunch of tests in `tests.cli_tests` are failing. They are all mock tests which verify that `load_config` was called with specific arguments. Each test needs to be updated by having the `page_type` argument added to it.